string frame_id
string object_type
string sub_type
---
bool result
geometry_msgs/Point position
